src/bin/playground.cpp

```diff
@@ -18,7 +20,10 @@
 static const int CHUNK_COUNT = 50;
 const auto COLUMN_COUNT = uint32_t{23};
 const auto ROW_COUNT = uint32_t{65'000};
+
 const auto FILENAME = "z_binary_test.bin";
+std::ofstream FILESTREAM;
```

Comment:
In the storage manager, this will surely be done in the writing method. For the sake of convenience, I'd keep it like this in the playground.

Reply:
The playground will be emptied by #103 and the filestreams are needed to remove the functions we copied from the binary_writer. The implementation for this issue (single filestream and removal of binary_writer functions) will be put on our issues list. This will then take place in the storage manager, as #100 moved the playground contents there.